docs/Performance and Tuning/ZFS on Linux Module Parameters.rst

```diff
@@ -1209,6 +1212,29 @@ legacy behavior of writing decompressed data to cache devices.
 |                   | design                                          |
 +-------------------+-------------------------------------------------+
 
+l2arc_meta_percent
+~~~~~~~~~~~~~~~~~~
+
+Limit ARC consumption by L2ARC-only headers at specified percentage.
```

Comment:
I propose to just duplicate description from man pages here, it may be more straightforward for newcomers. 

```suggestion
Percent of ARC size allowed for L2ARC-only headers.
Since L2ARC buffers are not evicted on memory pressure, too large amount of
headers on system with irrationaly large L2ARC can render it slow or unusable.
This parameter limits L2ARC writes and rebuild to achieve it.
```

Reply:
Done.  Though I find it waste of time to maintain duplicated documentation.